frontend/src/lib-components/employee/messages/MessageEditor.tsx

```diff
@@ -418,15 +425,39 @@ export default React.memo(function MessageEditor({
       </BottomBarMobile>
     </ContainerMobile>
   ) : (
-    <Container data-qa="message-editor" data-status={draftState}>
+    <Container
+      data-qa="message-editor"
+      data-status={draftState}
+      expanded={expandedView}
+    >
       <TopBar>
         <Title>{title}</Title>
-        <IconButton
-          icon={faTimes}
-          onClick={onCloseHandler}
-          white
-          data-qa="close-message-editor-btn"
-        />
+        <HeaderButtonContainer>
+          {expandedView ? (
+            <IconButton
+              icon={faDownLeftAndUpRightToCenter}
+              onClick={() => setExpandedView(!expandedView)}
```

Comment:
This could be a good exercise for the "memo everything" stuff we discussed at some point.

Define a new function:
```
const toggleExpandedView = useCallback(
  () => setExpandedView((prev) => !prev),
  []
)
```
and use it:
```
<IconButton ... onClick={toggleExpandedView} ... />
```

This way, `IconButton` doesn't have to re-renderer each time something else changes.

On the other hand, in this component it doesn't matter as much because there's a lot of other things causing re-rendering of other child components.